feature/gnpsi/otg_tests/sampling_test/README.md

```diff
@@ -0,0 +1,112 @@
+# gNPSI-1: Sampling and Subscription Check
+
+## Summary
+
+The goal is to validate that packet sampling is working as expected, clients can connect to the gNPSI service and receive samples.  
+
+## Procedure
+
+
+### Test 1: Verify samples are exported via gNPSI service. 
+
+1. Enable sFlow and gNPSI on the DUT.
+
```

Comment:
Need to add steps (including oc paths) for how to configure gNPSI.  Don't forget to include configuring certificate, etc:
[/system/grpc-servers/grpc-server/config/certificate-id](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-system.html#system-grpc-servers-grpc-server-config-certificate-id)

Reply:
@dplore perhaps grpc-server part of the config can be left out? For example, in other tests, we don't configure p4rt or gribi grpc services separately, it is assumed that they're included in the base config.

Reply:
Ah, this is true, there is an implicit requirement that the DUT already has those gRPC services enabled and configured.  Ok, we can follow that precedent here and not include this here.